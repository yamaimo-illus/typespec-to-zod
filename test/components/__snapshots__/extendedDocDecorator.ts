import { z } from "zod";

/*
 ██████╗ ██████╗ ███╗   ███╗██████╗  ██████╗ ███╗   ██╗███████╗███╗   ██╗████████╗███████╗
██╔════╝██╔═══██╗████╗ ████║██╔══██╗██╔═══██╗████╗  ██║██╔════╝████╗  ██║╚══██╔══╝██╔════╝
██║     ██║   ██║██╔████╔██║██████╔╝██║   ██║██╔██╗ ██║█████╗  ██╔██╗ ██║   ██║   ███████╗
██║     ██║   ██║██║╚██╔╝██║██╔═══╝ ██║   ██║██║╚██╗██║██╔══╝  ██║╚██╗██║   ██║   ╚════██║
╚██████╗╚██████╔╝██║ ╚═╝ ██║██║     ╚██████╔╝██║ ╚████║███████╗██║ ╚████║   ██║   ███████║
 ╚═════╝ ╚═════╝ ╚═╝     ╚═╝╚═╝      ╚═════╝ ╚═╝  ╚═══╝╚══════╝╚═╝  ╚═══╝   ╚═╝   ╚══════╝
*/
export const schemaExtendedSyntaxForDocDecorator = z.object({
    // Username
    name: z.string().min(2, { message: 'Must be 2 or more characters long' })
});