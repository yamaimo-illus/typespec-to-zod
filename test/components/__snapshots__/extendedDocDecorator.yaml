openapi: 3.0.0
info:
  title: (title)
  version: 0.0.0
tags: []
paths: {}
components:
  schemas:
    ExtendedSyntaxForDocDecorator:
      type: object
      required:
        - name
      properties:
        name:
          type: string
          description: "Username zod: z.string().min(2, { message: 'Must be 2 or more characters long' })"
